examples/server/server.cpp

```diff
@@ -724,6 +725,13 @@ static void server_params_parse(int argc, char **argv, server_params &sparams,
             }
             params.n_ctx = std::stoi(argv[i]);
         }
+        else if (arg == "-gqa" || arg == "--gqa") {
+            if (++i >= argc) {
+                invalid_param = true;
+                break;
+            }
+            params.n_gqa = std::stoi(argv[i]);
+        }
```

Comment:
The rest of this file places the opening braces on a new line, let's use the same everywhere for consistency.

Reply:
Fixed. Also changed a couple of lines nearby that also used K&R style.

Reply:
This will be changed in #2217 to the usual.